core/src/main/java/org/elasticsearch/cluster/metadata/IndexMetaData.java

```diff
@@ -219,6 +220,16 @@ public static State fromString(String state) {
     public static final Setting<Settings> INDEX_ROUTING_INITIAL_RECOVERY_GROUP_SETTING =
         Setting.groupSetting("index.routing.allocation.initial_recovery."); // this is only setable internally not a registered setting!!
 
+    /**
+     * The number of active shard copies to check for before proceeding with a write operation.
+     */
+    public static final Setting<ActiveShardCount> WAIT_FOR_ACTIVE_SHARDS_SETTING =
```

Comment:
It's a bit of mess already - so there are many conventions here - but let's try to follow of the majority here and name this `SETTING_WAIT_FOR_ACTIVE_SHARDS` ? also it will be good to open a follow up to discuss location of index settings and their naming conventions. It's all over the place now